pkg/network/ebpf/c/protocols/grpc/helpers.h

```diff
@@ -9,7 +9,7 @@
 #include "protocols/http2/skb-common.h"
 #include "protocols/grpc/defs.h"
 
-#define GRPC_MAX_FRAMES_TO_FILTER 10
+#define GRPC_MAX_FRAMES_TO_FILTER 90
```

Comment:
lets add documentation for both of the consts as we do in the decoding part.

Reply:
done